tests/extmod_hardware/machine_uart_irq_break.py

```diff
@@ -0,0 +1,67 @@
+# Test machine.UART.IRQ_BREAK firing after a break is received.
+#
+# IMPORTANT: This test requires hardware connections: the UART TX and RX
+# pins must be wired together.
+
+try:
+    from machine import UART
+
+    UART.IRQ_BREAK
+except (ImportError, AttributeError):
+    print("SKIP")
+    raise SystemExit
+
+import time, sys
+
+# Configure pins based on the target.
+if "esp32" in sys.platform and not (
+    "ESP32S2" in sys.implementation._machine or "ESP32C3" in sys.implementation._machine
+):
+    # The ESP32 needs separate UART instances for the test
```

Comment:
It's a shame that esp32 needs two instances... I see why, it's because sendbreak() changes the TX and RX baudrate.

I think we should change sendbreak() so it pulls the TX GPIO low for the correct period of time.  Otherwise in real applications one could lose RX chars while doing a sendbreak.  (But this is for a separate PR in the future.)

Reply:
OK